example/package.json

```diff
@@ -5,8 +5,8 @@
   "dependencies": {
     "@craco/craco": "^6.1.1",
     "@reach/router": "^1.3.4",
-    "@strapi/design-system": "^0.0.1",
-    "@strapi/icons": "^0.0.1",
+    "@strapi/design-system": "file:../packages/strapi-design-system",
+    "@strapi/icons": "file:../packages/strapi-icons/dist",
```

Comment:
as Hicham suggested some times ago, this is necessary to make sure we use the library the way it should be used.